tests/lcs/core/engine/send.livecodescript

```diff
@@ -71,4 +92,19 @@ on TestSendDeleteMe
 	send "--execute this" & return & "delete me" to it
 
 	TestAssert "execute 'delete me' using send", there is not a stack it
-end TestSendDeleteMe
\ No newline at end of file
+end TestSendDeleteMe
+
+on TestSendAsScript
+	local tStack
+	create stack
+	put it into tStack
+	set the defaultStack to the short name of tStack
+	create button
+	
+	local tSendScript
+	put "delete button 1 of me" into tSendScript
```

Comment:
@livecodeali should we test param evaluation in the object context? Maybe getting the value of a script local? Hmm... if we can do that we could add script locals to the SE when not debugging. That would be lovely.